GoedeDoelenHelpen/ClientApp/src/app/authentication.service.ts

```diff
@@ -16,6 +16,11 @@ export type SignUpModel = {
   profileimage: any;
 };
 
+export type LoginModel = {
```

Comment:
Why a new LoginModel instead of the excisting SignInModel?

Reply:
Omdat dat mijn oplossing was maar ik haal het eruit